instrumentation/jvm-metrics/src/main/java/com/splunk/opentelemetry/instrumentation/jvmmetrics/AllocatedMemoryMetrics.java

```diff
@@ -25,19 +25,18 @@
 
 public class AllocatedMemoryMetrics {
   public static final String METRIC_NAME = "process.runtime.jvm.memory.allocated";
-  private static final boolean hasComSunThreadMXBean = hasComSunThreadMXBean();
   private final AllocationTracker allocationTracker = createAllocationTracker();
 
-  public boolean isAvailable() {
-    return allocationTracker != null;
+  public boolean isUnavailable() {
```

Comment:
there are more usages of `isAvailable` in this module

Reply:
I wasn't trying for completeness, just the task at hand. I guess I can change those other unrelated ones as well to be consistent.